test/tests.js

```diff
@@ -41,6 +41,12 @@ QUnit.test('percent character in cookie value mixed with encoded values', functi
 	assert.strictEqual(Cookies.get('bad'), 'foo%bar"baz%bax=', 'should read the percent character');
 });
 
+QUnit.test('percent character in cookie value mixed with encoded values lower case', function (assert) {
+	assert.expect(1);
+	document.cookie = 'mybad=foo%bar%22baz%bax%3d';
```

Comment:
@mmilford Do you think we can improve the docs about this? Do you have any suggestion? Thanks!